app/controllers/performance_tracking_controller.rb

```diff
@@ -5,8 +5,11 @@ def index; end
 
   private
 
+  # Users that can manage others are only allowed to access this page on staging.
+  # This is configured by the allow_user_managers_service_access feature flag.
   def require_supervisor!
     return if current_user.supervisor? && FeatureFlags.basic_user_roles.enabled?
+    return if FeatureFlags.allow_user_managers_service_access.enabled?
 
     render status: :not_found, template: 'errors/not_found', layout: 'errors'
```

Comment:
Line 14 is never reached at the moment - is that intentional?

Reply:
What do you mean never reached? It is reached for caseworkers who shouldn't be able to see the page

Reply:
Ah sorry misread the statement!